src/components/Pill/Pill.tsx

```diff
@@ -43,7 +43,6 @@ export interface Props extends Omit<PillButtonProps, 'toggleDropdown' | 'childre
  */
 export const Pill: React.FC<Props> = ({
     onClear,
-    doneLabel,
```

Comment:
you remove it from here but you did not remove it from the props. Also the onClose button js docs are mentioning this done button.